src/core/public/doc_links/doc_links_service.ts

```diff
@@ -126,16 +127,16 @@ export class DocLinksService {
           introduction: `${OPENSEARCH_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/index-patterns.html`,
         },
         addData: `${OPENSEARCH_WEBSITE_URL}guide/en/kibana/${DOC_LINK_VERSION}/connect-to-elasticsearch.html`,
```

Comment:
do we want to replaced the "kibana" reference or just delete it to remove the reference or should we verify first if it's fine to leave the word kibana in?

Reply:
Yeah I can just remove the reference for now.